spec/models/status_spec.rb

```diff
@@ -439,6 +439,59 @@
     end
   end
 
+  describe '.permitted_for' do
+    subject { described_class.permitted_for(target_account, account).pluck(:visibility) }
+
+    let(:target_account) { alice }
+    let(:account) { bob }
+    let!(:public_status) { Fabricate(:status, account: target_account, visibility: 'public') }
```

Comment:
In my opinion, it is not needed because symbol value is cast to string with `ActiveRecord::Enum::EnumType` .

```
Status.new(visibility: :public).visibility.is_a?(String) #=> true
```